doc/classes/LookAtModifier3D.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="LookAtModifier3D" inherits="SkeletonModifier3D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		The [SkeletonModifier3D] rotates a bone to look a target.
+		The [LookAtModifier3D] rotates a bone to look a target.
```

Comment:
```suggestion
		The [LookAtModifier3D] rotates a bone to look at a target.
```
Same here

Reply:
Improved the wording there and on line 7, thank you